### ISSUE TITLE
Grey bar appears under menu

### ISSUE THREAD

**Issue by matisaul** [maykar/compact-custom-header#49]
**Describe the bug**
Grey bar appears under menu

**Config**
```
          - type: "custom:compact-custom-header"
            main_config: true
            header: true
            menu: show
            notification: overflow
            voice: clock
            options: show
            clock_format: 24
            background_image: true
            hide_tabs: 0,1,2,3,4,5,6,7,8,9,10
```

**Error**

**Screenshots**
https://imgur.com/dK6C9n7

**Version**
Home Assistant Version
0.87.1
Compact Custom Header Version
1.0.0b1
Browser type and version
Chrome 71.0.3578.98
**Additional context**

That grey bar appears when I load the page, but if I push menu button, or if I go to another page (Configuration Menu for example) and then back to lovelace, that grey bar dissapear. It happens on my Pc and Android mobile. For additional info, I'm hiding all my tabs.

**Comment by maykar** [comment]
Could you share your background image and share your theme code so I can test

**Comment by matisaul** [comment]
Sure,

My background: https://imgur.com/5T4wlyP

My theme:

```
material_dark_blue:
  light_blue: "#2581ab"
  dark_blue: "#16516B"
  dark_grey: "#232326"
  light_grey: "#404B51"
  red: "#e31d1d"
  yellow: "#fffa0c"
  white: "#f7f7f7"
  pink: "#ff0090"
  blue: "#48C072"
  green: "#3CF743"
  grey_transparent: "rgba(44, 44, 44, 0.3)"
  primary-color: "var(--light_blue)"
  light-primary-color: "var(--light_blue)"
  dark-primary-color: "var(--dark_blue)"
  text-primary-color: "var(--white)"
  primary-background-color: "var(--dark_grey)"
  sidebar-text_-_background: "var(--white)"
  paper-card-background-color: "var(--grey_transparent)"
  paper-card-header-color: "var(--white)"
  secondary-background-color: "var(--dark_grey)"
  primary-text-color: "var(--white)"
  secondary-text-color: "var(--white)"
  paper-listbox-background-color: "var(--light_grey)"
  paper-toggle-button-checked-ink-color: "var(--white)"
  paper-toggle-button-checked-button-color: "var(--yellow)"
  paper-toggle-button-checked-bar-color: "var(--yellow)"
  paper-slider-knob-color: "var(--yellow)"
  paper-slider-knob-start-color: "var(--red)"
  paper-slider-pin-color: "var(--white)"
  paper-slider-active-color: "var(--yellow)"
  paper-slider-secondary-color: "var(--white)"
  paper-slider-container-color: "var(--red)"
  paper-progress-secondary-color: "var(--white)"
  paper-item-selected_-_background-color: "var(--light_blue)"
  paper-item-icon_-_color: "var(--white)"
  paper-item-icon-color: "var(--light_blue)"
  paper-item-icon-active-color: "var(--yellow)"
  paper-grey-50: "var(--light_blue)"
  paper-grey-200: "var(--dark_blue)"
  label-badge-background-color: "var(--light_grey)"
  label-badge-text-color: "var(--white)"
  label-badge-red: "var(--red)"
  label-badge-blue: "var(--blue)"
  label-badge-green: "var(--green)"
  label-badge-yellow: "var(--yellow)"
  label-badge-grey: "var(--grey)"
  table-row-background-color: "var(--light_grey)"
  table-row-alternative-background-color: "var(--dark_grey)"

```

**Comment by maykar** [comment]
and what does your lovelace code look like, specifically the background styling

**Comment by matisaul** [comment]
`background: url("/local/images/lovelace/fondos/dark5.jpg") repeat`

But if I disable background, there are still this gap between menu bar and the cards. 
It dissapears if I go to another HA menu, and return to Lovelace.

Here is a gif from my web browser:
https://ibb.co/v3d9pfw

**Comment by maykar** [comment]
Ah okay, yeah seems unrelated to background image. So hiding tabs is more for disabling tabs for certain users. Hiding all the tabs might have unintended side effects, does this happen when the tabs aren't hidden?

If that is what's causing it, I can reintroduce a feature of the old card that didn't make it into the update that hides the tab container.

**Comment by maykar** [comment]
This should be fixed in the latest release 1.0.0b4

This was due to the card trying to redirect to the first visible tab, but since all were hidden it caused some issues. It now checks to see if all are hidden and if so will not redirect.

**Comment by matisaul** [comment]
Nice! Thanks!
